utilities/bash/run_six.sh

```diff
@@ -1596,7 +1649,7 @@ function treatLong(){
 	        	if [ "$sixdeskplatform" == "lsf" ] ; then
 	        	    dot_bsub
 			    local __subSuccess=$?
-	        	elif [ "$sixdeskplatform" == "htcondor" ] ; then
+	        	elif [ "$sixdeskplatform" == "htcondor" ] || [ "$sixdeskplatform" == "htboinc" ] ; then
```

Comment:
I would maybe split here the conditional statement, and create a small dot_htboinc function, where you echo in the .list file not only $Rundir, but also the name of the .zip file